docs/source/pillars/supporting.md

```diff
@@ -0,0 +1,217 @@
+(pillar_supporting)=
+
+# Supporting
+
+```{figure} ../../images/Capability_Map/full.drawio.svg
+:alt: SATRE Pillars Capability Map
+:align: center
+
+SATRE Pillars Capability Map
+```
+
+<!-- See all pillars of the SATRE Pillars Capability Map here: {ref}`satre_pillars` -->
+
+## Business continuity management
+
+_What the organisation does to ensure the development, testing, and maintenance of business continuity plans._
+
+```{list-table}
+:header-rows: 1
+:name: tab-business-continuity-subject
+
+* - Statement
+  - Guidance
+  - Importance
+* - You have a business continuity plan that includes consideration of loss of service for deployed TREs
+  - This may be due to downtime from service providers, a breach, or loss of power. Your plan should detail your process for managing loss of service for deployed TREs, and evaluation of impact of such loss
+  - Recommended
+* - You regularly test the aspects of your business continuity plan concerning TREs, and have a process in place to iterate the plan if required
+  -
+  - Recommended
+```
+
+## Project and programme management
+
+_What the organisation does to ensure effective management of programmes and projects._
+
+```{list-table}
+:header-rows: 1
+:name: tab-project-programme-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - All projects requiring TRE are assigned project management support
```

Comment:
"project management" can mean a lot of things (as can "project"). Is there another term we can use, since most research projects will already have their own project manager?